democracylab/settings.py

```diff
@@ -169,6 +169,7 @@
 
 USE_TZ = True
 
+PROJECTS_PER_PAGE = 2
```

Comment:
Let's make this configurable

Reply:
OH! I forgot to reset that to 10.   What do you mean/intend by "configurable"?  Who configures it and how?

Reply:
By environment variable.  If you check the rest of the settings file, you'll see os.environ.get() lines where we read them in.

Reply:
OK. Got that working. Slacked you an updated .sh file with projects per page included.